requirements.txt

```diff
@@ -1,2 +1,5 @@
 pluggy
 setuptools
+opentelemetry-api
+opentelemetry-sdk
+opentelemetry-exporter-otlp
```

Comment:
I think this dependency can be removed now, because we now expect that the implementer of the otel_exporter hook (e.g. pub) is the one which declares the dependency on this (or on whatever exporter it wants to use).

Reply:
Removed opentelemetry-exporter-otlp.